draft-ietf-tls-dtls13.md

```diff
@@ -2088,6 +2092,9 @@ registry, defined in {{!TLS13}}, for RequestConnectionId (TBD), and
 NewConnectionId (TBD), as defined in this document.  The value for the
 "DTLS-OK" columns are "Y".
 
+IANA [SHALL mark/has marked] the cipher suites TLS_SHA256_SHA256 and
+TLS_SHA384_SHA_384 DTLS=N.
```

Comment:
```suggestion
IANA [SHALL set] the "DTLS-OK" column value for cipher suites TLS_SHA256_SHA256 and
TLS_SHA384_SHA_384 to "N".
```

Reply:
Why do we need to add this IANA entry in this document? Shouldn't https://tools.ietf.org/html/draft-camwinget-tls-ts13-macciphersuites-08 add it to IANA given that they are defining the ciphersuite?

Reply:
I agree with Hannes; I don't think we need to do anything about the MAC-only ciphersuites entries.
I've already been in contact with the authors (and DEs) and the authors plan to add text to handle this case.  (Though, they claim that the record numbers will just not be encrypted and that's okay.)

Reply:
They are already in the registry and marked DTLS-OK.

Reply:
Yes, they are.  Is that wrong?  Is that more wrong than having MAC-only ciphersuites in the first place?

Reply:
Yeah, I guess we have to live with this.